planning/.pages

```diff
@@ -76,9 +76,6 @@ nav:
     - 'RTC Interface': planning/rtc_interface
   - 'Additional Tools':
     - 'RTC Replayer': planning/rtc_replayer
-    - 'Planning Debug Tools':
-      - ['About Planning Debug Tools'](https://github.com/autowarefoundation/autoware_tools/tree/main/planning_debug_tools)
-      - ['Stop Reason Visualizer'](https://github.com/autowarefoundation/autoware_tools/tree/main/planning_debug_tools/doc-stop-reason-visualizer)
```

Comment:
Only with these 3 lines removed, the CI has passed:
- https://github.com/autowarefoundation/autoware.universe/actions/runs/8084445806/job/22089737715?pr=6509

Now I will try to find a compromise.